sigmf-spec.md

```diff
@@ -238,7 +237,17 @@ capture segment objects:
 |----|--------------|-------|-----------|
 |`sample_start`|true|uint|The sample index at which this segment takes effect.|
 |`frequency`|false|double|The center frequency of the signal in Hz.|
-|`time`|false|string|An ISO-8601 formatted string indicating the timestamp of the sample index specified by `sample_start`.|
+|`datetime`|false|string|An ISO-8601 string indicating the timestamp of the sample index specified by `sample_start`. More details, below.|
+
+###### The `datetime` Pair
+
+This name/value pair must be an ISO-8601 string of the form
+`YYYY-MM-DDTHH:MM:SS.SSSZ`, where the `T` is the delimiter between the date and
+the time, and the `Z` indicates the UTC/Zulu timezone. All timestamps must be in
```

Comment:
Defining the meanings of T and Z seems beside the point whereas the clarification that is most needed (for anyone who doesn't already find it obvious) is that `T` and `Z` are literal text. Maybe write it out with markup for metasyntactic elements, like <code><var>YYYY</var>-<var>MM</var>-<var>DD</var>T<var>HH</var>:<var>MM</var>:<var>SS</var>.<var>SSS</var>Z</var></code>.

    This name/value pair must be an ISO-8601 string of the form
    <code><var>YYYY</var>-<var>MM</var>-<var>DD</var><!--
    -->T<var>HH</var>:<var>MM</var>:<var>SS</var>.<var>SSS</var>Z</var></code>.
    All timestamps must be in the UTC/Zulu timezone, and use the 24-hour clock.